3581/3/app/controllers/places_controller.rb

```diff
@@ -0,0 +1,22 @@
+class PlaceController < ApplicationController
+  get '/' do
+    @places = Place.all
+    erb :index
+  end
+
+  get '/place/:id' do
+    @place = Place.find(params[:id])
+    @review = Review.where(place_id: params[:id]).reverse
```

Comment:
```ruby
@reviews would be better
```
and try
```ruby
@place.reviews.reverse
```
please, read the guide https://guides.rubyonrails.org/association_basics.html for better understanding what you can do with ActiveRecord